subsys/bluetooth/host/iso.c

```diff
@@ -1238,7 +1238,7 @@ void hci_le_cis_req(struct net_buf *buf)
 	}
 
 	/* Lookup ACL connection to attach */
-	acl = bt_conn_lookup_handle(acl_handle);
+	acl = bt_conn_lookup_handle(acl_handle, BT_CONN_TYPE_ALL);
```

Comment:
Should probably be `BT_CONN_TYPE_LE`. @Thalley can you confirm?

Reply:
Yes, a "parent" connection to a CIS is always a `BT_CONN_TYPE_LE`